compiler/optimizer/InductionVariable.cpp

```diff
@@ -3709,20 +3714,11 @@ bool TR_LoopStrider::reassociateAndHoistComputations(TR::Block *loopInvariantBlo
 
          TR::SymbolReference *internalPointerSymRef = (*_reassociatedAutos)[originalInternalPointerSymbol];
          originalNode->getFirstChild()->recursivelyDecReferenceCount();
-         node->decReferenceCount();
```

Comment:
What is the rationale for these changes ? This seems to be unrelated to what was discussed in the commit message ?

Reply:
From what I understood, refcount of node (offset node) was being decremented because we were replacing it with it's second child. I think that was just carried over from array access trees with array header size because we don't need to recalculate element offset after hoisting dataAddr node in off-heap mode. I removed the decrement refcount logic as part of my fix for [2nd bug](https://github.com/eclipse-omr/omr/pull/7509#issuecomment-2447431084).